src/components/views/dialogs/ForwardDialog.tsx

```diff
@@ -216,6 +216,7 @@ const ForwardDialog: React.FC<IProps> = ({ matrixClient: cli, event, permalinkCr
         },
         event_id: "$9999999999999999999999999999999999999999999",
         room_id: event.getRoomId(),
+        origin_server_ts: event.getTs(),
```

Comment:
It's not necessarily a bad thing, but I'm curious about why we've added so many `origin_server_ts`es to mock events. I can't see any change that would require it?